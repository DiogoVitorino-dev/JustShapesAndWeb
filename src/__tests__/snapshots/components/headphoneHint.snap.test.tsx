import { render } from "@testing-library/react-native";

import HeadphoneHint from "@/components/menu/headphoneHint";

describe("Headphone Hint - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(<HeadphoneHint />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
