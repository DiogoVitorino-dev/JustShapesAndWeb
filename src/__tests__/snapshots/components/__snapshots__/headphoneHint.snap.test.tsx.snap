// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headphone Hint - Snapshot component test should render correctly 1`] = `
<View
  collapsable={false}
  jestAnimatedStyle={
    {
      "value": {},
    }
  }
  style={
    {
      "alignItems": "center",
      "backgroundColor": "rgb(3, 38, 25)",
      "flex": 1,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <View
    collapsable={false}
    jestAnimatedStyle={
      {
        "value": {},
      }
    }
    style={
      {
        "opacity": 0,
        "transform": [
          {
            "scale": 1,
          },
          {
            "translateY": 0,
          },
        ],
      }
    }
  >
    <Text />
  </View>
  <Text
    collapsable={false}
    jestAnimatedStyle={
      {
        "value": {},
      }
    }
    style={
      {
        "color": "rgb(245, 245, 245)",
        "fontFamily": "Manjari",
        "fontSize": 22,
        "marginBottom": -3,
        "opacity": 0,
        "transform": [
          {
            "translateY": -0,
          },
        ],
      }
    }
  >
    Use fones de ouvido para melhor experiÃªncia
  </Text>
  <View
    collapsable={false}
    style={
      [
        undefined,
        [
          {
            "height": "100%",
            "position": "absolute",
            "width": "50%",
          },
          {
            "width": "100%",
          },
        ],
      ]
    }
  >
    <View
      collapsable={false}
      jestAnimatedStyle={
        {
          "value": {
            "left": 0,
            "opacity": 0,
            "top": 0,
            "transform": [
              {
                "scale": 1,
              },
            ],
          },
        }
      }
      style={
        {
          "borderColor": "rgb(20, 210, 255)",
          "borderRadius": 0,
          "borderWidth": 3,
          "height": 0,
          "left": 0,
          "opacity": 0,
          "top": 0,
          "transform": [
            {
              "scale": 1,
            },
          ],
          "width": 0,
          "zIndex": 1000,
        }
      }
    />
  </View>
</View>
`;
